resources/application-connector/charts/application-registry/templates/role-binding.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if not .Values.global.disableLegacyConnectivity }}
```

Comment:
Just a question
Wouldn't it be easier to use `requirements.yaml` in case of disabling whole charts?

Reply:
Well, I was thinking about it. It is possible for Application Registry but not for other components (e.g. Connector Service). I though it would be easier to comprehend when done this way, but I'm open to discussion.